### ISSUE TITLE
[BUG] Player Deployables are not automatically created on mech import

### ISSUE THREAD

**Issue by DCShank** [Eranziel/foundryvtt-lancer#738]
**Describe the bug**
When a player creates their pilot and syncs with Comp/Con to import their mech, instead of automatically creating actors for their deployables, it sends messages in the chat visible to both the GM and the Player with a button that says "IMPORT [name of deployable] FOR [name of mech]?". Clicking the button creates the deployable.

**To Reproduce**
1. Configure Players to be allowed to create actors
2. Create a Player
3. Login as the Player (Remaining steps are from the players user)
4. Login to Comp/Con
5. Create a Pilot Actor
6. Sync your pilot to a mech in Comp/Con

**Expected behavior**
The mech and deployables are automatically imported from Comp/Con

**Screenshots**
![image](https://github.com/user-attachments/assets/6f227eac-dba7-4a91-a737-e793a6bcc218)

**Platform (please complete the following information):**
 - OS: Linux (Manjaro Linux 24.0.4)
 - Browser Firefox
 - Foundry version  11.315
 - Lancer system version 2.1.8

**Additional context**

**Discord contact info**
dcshank

**Comment by Eranziel** [comment]
That is intentional. It circumvents a few tricky bugs that can happen when player clients try to create actors.

I'll leave this issue open for a bit in case someone else on the team wants to tackle it, but this is probably a won't-fix

**Comment by BoltsJ** [comment]
What's different for the deployables compared to mechs? I was under the impression that mechs get created just fine?

**Comment by DCShank** [comment]
Yeah, mechs seem to get imported fine and I'm not sure what the difference would be, although I'm in no way an expert on foundry module development
